src/components/SearchResult.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component } from 'react';
+import { ISearchResult } from '../types';
+
+export default class SearchResult extends Component<ISearchResult> {
+  render() {
+    const { data } = this.props;
+    return (
+      <div className="col">
+        {data.map((item) => (
+          <div className="card" key={item.uid}>
+            <b>{item.title}</b>
+            <ul>
+              <li>
+                {item.numberOfPages
+                  ? `Number of pages: ${item.numberOfPages}`
+                  : ''}
+              </li>
```

Comment:
Probably it would be better to hide `li` tag a well if there is no number of pages?
![image](https://github.com/golubeva-webmaster/react-st/assets/55094494/1f03fbf8-7f4f-40e9-91a6-1ead07f6128d)
```suggestion
              {item.numberOfPages && 
                <li>
                  {`Number of pages: ${item.numberOfPages}`}
                </li>
              }
```